### PR TITLE
Update image to 1.1.4

### PR DESCRIPTION
**Pull request by cdrage**
**What type of PR is this?**
> Uncomment only one ` /kind` line, and delete the rest.
> For example, `> /kind bug` would simply become: `/kind bug`

/kind feature

**What does does this PR do / why we need it**:

Updates the image to version 1.1.4 of odo-init-image

**Which issue(s) this PR fixes**:

N/A

**PR acceptance criteria**:

N/A

**How to test changes / Special notes to the reviewer**:

N/A

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

File: pkg/devfile/adapters/common/utils.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ const (
 
 	// Default Image that will be used containing the supervisord binary and assembly scripts
 	// use GetBootstrapperImage() function instead of this variable
-	defaultBootstrapperImage = "registry.access.redhat.com/openshiftdo/odo-init-image-rhel7:1.1.3"
+	defaultBootstrapperImage = "registry.redhat.io/ocp-tools-4/odo-init-container-rhel8:1.1.4"
```
Comment (amitkrout) [inline pkg/devfile/adapters/common/utils.go:35]: Curious to know how authentication will be handled during the image pull ?

**Comment by amitkrout** [comment]
platform issue
/retest

File: pkg/devfile/adapters/common/utils.go (hunk 1 above)
Comment (amitkrout) [reply pkg/devfile/adapters/common/utils.go:35]: Travis failure
``` 
✗  Pulling image registry.redhat.io/ocp-tools-4/odo-init-container-rhel8:1.1.4 [664ms]
```

This what i was expecting, but 4.2 and 4.5 confused me by passing the same test 😕

Comment (amitkrout) [reply pkg/devfile/adapters/common/utils.go:35]: Travis failure
```
[odo]  ✗  Waiting for component to start [4m] [WARNING x5: Failed]

```

Comment (amitkrout) [reply pkg/devfile/adapters/common/utils.go:35]: /hold

**Comment by cdrage** [comment]
@amitkrout @mohammedzee1000 

When we released the image it was to an authenticated registry. We will have to wait until it's pushed to an unauthenticated registry so users may pull the image.

**Comment by cdrage** [comment]
Closing in favour of 1.1.5 update.

**Pull request closed without merging**